src/components/CounterTabs/CounterTabs.js

```diff
@@ -2,17 +2,17 @@ import React from 'react';
 import CounterIcon from '../CounterIcon';
 
 
-const CounterTabs = ({tabClick, heroID, currentTab}) => {
-    console.log(`HERO: ${heroID}\n CURRENT" ${currentTab}\n`);
+const CounterTabs = ({tabClick, heroID, currentTab, counter}) => {
+
```

Comment:
consider destructuring `icon_url` off `counter`, right here. For example, it would allow you to easily refactor the JSX into its own function, where it takes `icon_url` as a parameter, simply by copy/pasting the JSX, and then not need to delete the `counter` prefix everywhere. Plus, it clearly declares what properties you're going to be using, below.